### ISSUE TITLE
Useless use of '\'; doesn't escape metacharacter '['

### ISSUE THREAD

**Issue by tom-mi** [sanko/template-liquid#1]
When using the Template::Liquid module, I get the following warning:

```
Useless use of '\'; doesn't escape metacharacter '[' at /path/to/lib/perl5/Template/Liquid/Utility.pm line 9.
```

**Comment by latk** [comment]
Nope, this bug is real. It's no false positive at all. It was fixed by your PR #3, but the fixed version has yet to be released to CPAN. Could you  ( @sanko ) please do that?

Today I came across this issue when trying out `Template::Liquid` v1.0.2 under a very stable perl v5.20.1.

**Comment by sanko** [comment]
Pushing to PAUSE. ...forgot.
